app/src/main/java/net/aiscope/gdd_app/ui/sample_preparation/SamplePreparationMapper.kt

```diff
@@ -27,6 +28,22 @@ object SamplePreparationMapper {
         }
     }
 
+    private fun getBloodType(bloodQualityValue: String, context: Context): BloodQuality {
```

Comment:
"BloodType" to me sounds like, you know, A+ or something. Since we call it 'BloodQuality' everywhere else, can we name the method 'getBloodQuality' too?

Reply:
@jbors see now

Reply:
👍